### PR TITLE
Add the ticket forms resource and unit tests

### PR DESCRIPTION
**Pull request by joseconsador**
Add the ticket forms resource and unit tests
/cc @miogalang @jwswj @mmolina @atroche

### CODE REVIEW

**Comment by joseconsador** [comment]
WIP 
depends on https://github.com/zendesk/zendesk_api_client_php/pull/107

**Comment by joseconsador** [comment]
:haircut: 

@miogalang @atroche @jwswj @mmolina

**Comment by miogalang** [comment]
:+1:

**Comment by jwswj** [comment]
Looks like this needs to be rebased

**Comment by joseconsador** [comment]
:haircut:

File: src/Zendesk/API/Resources/TicketForms.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        parent::setUpRoutes();
+
+        $this->setRoutes([
+            'clone'   => 'ticket_forms/{id}/clone.json',
+            'reorder' => 'ticket_forms/reorder.json'
+        ]);
+    }
+
+    /**
+     * Clones an existing ticket form (can't use 'clone' as method name)
```
Comment (jwswj) [inline src/Zendesk/API/Resources/TicketForms.php]: This kinda sucks

**Comment by jwswj** [comment]
:+1:

**Pull request merged**